snapcraft.yaml

```diff
@@ -919,6 +914,18 @@ parts:
       - bin/tar2sqfs
       - lib/libsquashfs.so*
 
+  vim:
```

Comment:
adding vim does not appear to be related to commit message

Reply:
It was a mistake to reintroduce a `vim` part as that was already dropped from `5.0-edge` by https://github.com/canonical/lxd-pkg-snap/pull/251